hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java

```diff
@@ -1547,6 +1547,32 @@ public boolean recoverTrash(String volumeName, String bucketName,
     return recoverResponse.getResponse();
   }
 
+  @Override
+  public long prepareOzoneManager(long flushWaitTimeout,
+                                  long flushCheckInterval) throws IOException {
+    Preconditions.checkArgument(flushWaitTimeout > 0,
+        "flushWaitTimeout has to be > zero");
```

Comment:
Should we add this check in `PrepareSubCommand` as well to give the user a specific error message if they pass a bad value? I'm not sure how the client handles these precondition generated exceptions and presents them to the user.

Reply:
Changed this to match the command's user parameter name.